server/src/controllers/AchievementsController.js

```diff
@@ -0,0 +1,37 @@
+const Achievement = require('../models/Achievement');
+const replaceDriveURL = require('../utility/replaceDriveURL');
+
+module.exports = {
+  async createAchievement(req,res){
+    try{
+      const body = req.body;
+      let imageUrl = "";
+      if (!!body['Achievement Image'] && body['Achievement Image'].length > 0) {
```

Comment:
We are sure Achievement Image is not going to be null, since we made it compulsory in the form, so we can remove this check.